spinalcordtoolbox/deepseg/models.py

```diff
@@ -77,6 +78,24 @@
         "contrasts": ["t2"],
         "default": False,
     },
+    "model_seg_ms_sc_mp2rage": {
+        "url": [
+            "https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20211223/model_seg_ms_sc_mp2rage.zip"
+        ],
+        "description": "Segmentation of spinal cord on MP2RAGE data from MS participants.",
+        "contrasts": ["mp2rage"],
+        "default": False,
+    },
+    "model_seg_ms_lesion_mp2rage": {
+        "url": [
+            "https://github.com/ivadomed/model_seg_ms_mp2rage/releases/download/r20211223/model_seg_ms_lesion_mp2rage.zip"
+        ],
+        "description": "Segmentation of multiple sclerosis lesions on cropped MP2RAGE spinal cord data. To crop the "
+                       "data you can first segment the spinal cord using the model 'model_seg_ms_lesion_mp2rage' and "
+                       "then crop the MP2RAGE image using 'sct_crop_image -i IMAGE -m IMAGE_seg'",
+        "contrasts": ["mp2rage"],
```

Comment:
With the addition of `mp2rage` as a contrast, we might want to also add `mp2rage` to the `"choices"` for the `-c` option of `sct_deepseg`?

Reply:
hum, good thinking. I have to say, I don't really know what will be the use of `-c` for now. With `sct_deepseg_sc` we had that flag because it was used to select (implicitly) the model. But the philosophy with `sct_deepseg` is that the user selects the model, which explicitly mentions the contrast. So maybe we can get rid of `-c` (for now) 🤷‍♂️ ?

Reply:
Very good points. :)